python/3d/auto_generated/symbols/qgsline3dsymbol.sip.in

```diff
@@ -107,12 +107,12 @@ Returns whether the renderer will render data with simple lines (otherwise it us
 Sets whether the renderer will render data with simple lines (otherwise it uses buffer)
 %End
 
-    QgsAbstractMaterialSettings *material() const;
+    QgsAbstractMaterialSettings *materialSettings() const;
```

Comment:
This is all an api break, it's not permitted

Reply:
All those header files have this warning:

```
\warning This is not considered stable API, and may change in future QGIS releases. It is
 * exposed to the Python bindings as a tech preview only.
```

Reply:
Ok, fair enough